skrf/__init__.py

```diff
@@ -43,7 +43,7 @@
 from .constants import * 
 from .taper import * 
 from .instances import *
-from .vectorFitting import *
+# from .vectorFitting import *
```

Comment:
by doing so, the `VectorFitting` class is no more available from the root. Is it on purpose?